src/EnergyPlus/EconomicTariff.cc

```diff
@@ -3980,7 +3980,9 @@ void LEEDtariffReporting(EnergyPlusData &state)
                 state,
                 s_orp->pdchLeedEtsDemUnt,
                 "District Cooling",
-                format("{}{}", convDemandStrings[(int)distCoolUnits], demandWindowStrings[(int)distCoolDemWindowUnits]));
+                format("{}{}",
+                       convDemandStrings[(int)distCoolUnits],
+                       (distCoolDemWindowUnits == DemandWindow::Invalid) ? "" : demandWindowStrings[(int)distCoolDemWindowUnits]));
```

Comment:
Optional: also protect here, like we do for gas?

Reply:
Yeah why not.